main.go

```diff
@@ -138,7 +152,16 @@ func main() {
 
 	}
 
-	cmd := exec.Command(registryBinary, command, registryConfig)
+	// run /bin/create-bucket
+	cmd := exec.Command("/bin/create-bucket")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if err := cmd.Run(); err != nil {
+		log.Fatal("Error starting the registry", err)
+	}
+	log.Println("INFO: registry started.")
```

Comment:
I feel these log messages are misleading here as we are just trying to create a bucket here.